src/App.js

```diff
@@ -23,6 +23,8 @@ function App() {
   const [postBody, setPostBody] = useState("")
   const [openModal, setOpenModal] = useState(false)
   const [loginStatus, setLoginStatus] = useState("")
+  const [registerStatus, setRegisterStatus] = useState("")
```

Comment:
we could use boolean for registerStatus as false at first and set it to true if response is false. But this works too.